environments/prod/prod.tfvars

```diff
@@ -1146,6 +1146,20 @@ frontends = [
       },
     ]
   },
+  {
+    name             = "fact"
+    mode             = "Prevention"
+    custom_domain    = "find-court-tribunal.service.gov.uk"
+    backend_domain   = ["firewall-prod-int-palo-prod.uksouth.cloudapp.azure.com"]
+    certificate_name = "find-court-tribunal-service-gov-uk"
```

Comment:
cert needs to be in vault for this to be merged

Reply:
Cert is in vault.